### ISSUE TITLE
[FEATURE REQUEST, RR2] Allow merging two config files

### ISSUE THREAD

**Issue by hugochinchilla** [spiral/roadrunner#625]
In roadrunner 1.x we could have two different config files, one for production and other for develpment.

An example development config file looked like this:

```
include:
  - .rr.yaml

reload:
  interval: 1s
  patterns: [".php"]
```

Now the include block is not handled as `spf13/viper` does not have this feature implemented, (but is requested https://github.com/spf13/viper/pull/893).

As a workaround for this problem I can suggest two solutions:

 * Allow passing `--config` multiple times on the command line and merge the configuration from multiple files.
 * Add a key `enabled` under reload, so I can call the server with `-o reload.enabled=false` in production.

**Comment by wolfy-j** [comment]
FYI, we implemented this feature in RR 1 itself. Not via Viper.

**Comment by hugochinchilla** [comment]
I would like having an `enabled` option in every plugin just because of the symmetry. This on it's own would suffice to have `reload` disabled in production.

But I would also like to be able to have very different logging configurations in each environment and this can be cumbersome with the `-o` options, so having an include mechanism would be the optimal solution.

**Comment by 48d90782** [comment]
@hugochinchilla Yeah, I agree to include `enabled` keys to every plugin. As for the 'includes', let's wait for some time for the `viper` ticket to get merged (let's say 1-2 weeks) and then return to the discuttion. I plan it to `2021-05-11` release cycle.
